packages/a11y/src/Skiplink.spec.tsx

```diff
@@ -0,0 +1,15 @@
+import { cleanup, render } from "@testing-library/react"
+import * as React from "react"
+import { SkipLink } from "./SkipLink"
+
+afterEach(cleanup)
```

Comment:
FYI you don't need to do this anymore: https://kentcdodds.com/blog/common-mistakes-with-react-testing-library#using-cleanup

Reply:
TIL - thanks!